rdmalib/include/rdmalib/connection.hpp

```diff
@@ -10,16 +10,17 @@
 #include <vector>
 #include <optional>
 
-#ifdef USE_LIBFABRIC
+//#ifdef USE_LIBFABRIC
```

Comment:
As above - why remove `ifdef`?

Reply:
see note above. Is there a way around that issue?